root/layout/components/Footer.js

```diff
@@ -1,95 +1,520 @@
+                >
+                  {l('Cover Art Archive')}
+                </a>
+              </li>
 
-        {nonEmpty(stash.last_replication_date) ? (
-          <>
-            <br />
-            {texp.l('Last replication packet received at {datetime}', {
-              datetime: $c.user
-                ? formatUserDate($c, stash.last_replication_date)
-                : stash.last_replication_date,
-            })}
-          </>
-        ) : null}
-      </p>
-    </div>
+            </ul>
+          </div>
+          <div className="col-sm-12 col-md-2">
+            <h3 className="fs-3 fw-bold color-black ms-3">
+              {l('Useful Links')}
+            </h3>
+            <ul className="list-style-type">
+              <li>
+                <a
+                  className="fw-bold fs-5"
+                  href="https://metabrainz.org/donate"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Donate')}
+                </a>
+              </li>
+              <li>
+                <a
+                  className="fw-bold fs-5"
+                  href="https://wiki.musicbrainz.org/Main_Page"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Wiki')}
+                </a>
+              </li>
+              <li>
+                <a
+                  className="fw-bold fs-5"
+                  href="https://community.metabrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Community')}
+                </a>
+              </li>
+              <li>
+                <a
+                  className="fw-bold fs-5"
+                  href="https://blog.metabrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Blog')}
+                </a>
+              </li>
+              <li>
+                <a
+                  className="fw-bold fs-5"
+                  href="https://www.redbubble.com/people/metabrainz/shop"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Shop')}
+                </a>
+              </li>
+              <li>
+                <a
+                  className="fw-bold fs-5"
+                  href="https://metabrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('MetaBrainz')}
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div className="row mt-4">
+          <div className="col-md-3 border-top pt-4 d-none d-md-block fs-5">
+            <p>
+              {l('OSS Geek?')}
+              {' '}
+              <a
+                href="https://github.com/metabrainz/musicbrainz-server"
```

Comment:
Linking to a decent how to contribute guide might be better than linking to the Github directly. That said, I'm not sure we have anything worth linking at the moment (https://musicbrainz.org/doc/Development/Introduction seems pretty pointless)

Reply:
I think the contribution guide should be included in the readme of the repository but linking the repository here is a must since it provides a quick access for the developers and not just first time visitors.